Entities/StudentNumber.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FRITeam.Swapify.Entities
+{
+    public class StudentNumber : BaseEntity
```

Comment:
What is it? Tha class name is one of the most important thinks...
It is not student number... Give better name and based on it rename everythink

Reply:
What about "personal number" or "student ID"

Reply:
StudentNumber - class name